gulpfile.js

```diff
@@ -411,3 +431,32 @@ function repeat(str, num) {
     }
     return resp;
 }
+
+
+// Controllers inheritance configs
+var ctrlConfig = {
+    ctrlRegex: /.*\.controller\.js$/,
+    ctrlNameRegex: /\.controller\(['"]([^'"]+)['"]/,
```

Comment:
let's try to not resorting to the use of hungarian notation.  

it looks like we are building a Controller JSON object, a better name might be `Controller` or a noun that is descriptive of the object we want to assign configuration data to.